.circleci/config.yml

```diff
@@ -246,7 +246,7 @@ commands:
       - run:
           name: Trigger generated code validation
           command: |
-            python3 scripts/ci-circleci-start-pipeline.py --token ${MOBILE_METRICS_TOKEN} --origin-slug mapbox/mapbox-maps-android --target-slug mapbox/mapbox-maps-android-internal --current-branch ${CIRCLE_BRANCH}
+            python3 scripts/ci-circleci-start-pipeline.py --token ${MOBILE_METRICS_TOKEN} --origin-slug mapbox/mapbox-maps-android --target-slug mapbox/mapbox-maps-android-internal --current-branch ${CIRCLE_BRANCH} pixel_match_public True
```

Comment:
should it be `--pixel_match_public True`. if its an argument?

Reply:
nice catch